README.md

```diff
@@ -5,3 +5,62 @@
 [![Build Status](https://travis-ci.org/lefticus/cpp_starter_project.svg?branch=master)](https://travis-ci.org/lefticus/cpp_starter_project)
 
 [![Build status](https://ci.appveyor.com/api/projects/status/ro4lbfoa7n0sy74c/branch/master?svg=true)](https://ci.appveyor.com/project/lefticus/cpp-starter-project/branch/master)
+
+
+## Getting Started
+
+
+    $ mkdir <project_directory_name>
+    $ cd <project_directory_name>
+    $ git init
+    $ git pull git@github.com:lefticus/cpp_starter_project
+
+At this point, you have a copy of the repository, with no remotes set up.
+You will probably want to start a new repository using https://github.com/new, 
+and add a remote:
+
+    $ git remote add origin https://github.com/<user>/<your_new_repo>.git
+
+If you know you're not going to use one or more of the optional gui/graphics 
+frameworks (fltk, gtkmm, imgui, etc.), you can remove them with `git rm`:
+
+    $ git rm -r src/<unnecessary_framework>
+
+## Necessary Dependencies
+
+1. A C++ compiler that supports C++17 (gcc 7+, clang 6+, Visual Studio 2019)
```

Comment:
I'm not certain which compilers are actually required for this project; this info is coming from https://en.cppreference.com/w/cpp/compiler_support. In any case, there should probably be separate CI builds for all the compilers listed here, and currently there are not.

Reply:
We do need to enhance the CI, but this looks good right now.

Reply:
I recently learned that I was misreading the Visual Studio version numbers, and that Visual Studio 2017 (version 15.8 with compiler 19.15) actually fully supports C++17, so I'm going to change the version number on that dependency accordingly.